core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class DeleteOptions {
   private boolean mRecursive;
+  private boolean mRemoveUFSFile;
```

Comment:
In our production environment, hard to avoid operation that insert overwrite in hdfs. So, ufs's file will inconsistent with alluxio, to fix this situation, we need to umount the path and mount again, and then load the data again.